homework-submission/barbarosapp/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import './App.css';
+import logo from './logo.svg';
+import Item from './staticToDo';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+        </header>
+        <section className="ToDoItem">
+          <Item
+            name="Graduation Thesis"
+            deadLine="20-10-2018"
+            check="Undone" />
+        </section>
+        <section className="ToDoItem">
+          <Item
+            name="Payment for Credit Card"
+            deadLine="10/10/2018"
+            check="Done" />
+        </section>
+        <section className="ToDoItem">
+          <Item
+            name="Renovation of House"
+            deadLine="01/12/2018"
+            check="Undone" />
+        </section>
+      </div>
+    );
+  }
+}
+
```

Comment:
You could save the data in an Array of objects or a json file and use map on it to make it more dynamic, what would you do if you had more than 10 tasks?

Also it makes sense to use a list here